Source/gendung.h

```diff
@@ -20,14 +22,54 @@ enum dungeon_type : int8_t {
 	DTYPE_NONE = -1,
 };
 
+enum lvl_entry : uint8_t {
+	ENTRY_MAIN,
+	ENTRY_PREV,
+	ENTRY_SETLVL,
+	ENTRY_RTNLVL,
+	ENTRY_LOAD,
+	ENTRY_WARPLVL,
+	ENTRY_TWARPDN,
+	ENTRY_TWARPUP,
+};
+
+enum {
+	// clang-format off
+	DLRG_HDOOR     = 0x01,
+	DLRG_VDOOR     = 0x02,
+	DLRG_CHAMBER   = 0x40,
+	DLRG_PROTECTED = 0x80,

```

Comment:
Woudn't it be less error prone/more readable to use bitshifting syntax for initializing flag enums?

```suggestion
	DLRG_HDOOR     = 1 << 0,
	DLRG_VDOOR     = 1 << 1,
	DLRG_CHAMBER   = 1 << 2,
	DLRG_PROTECTED = 1 << 3,
```

Reply:
may well be, i just moved it.